### ISSUE TITLE
Adep change along isotherm (Eq.(6))

### ISSUE THREAD

**Issue by mostafa-razavi** [mostafa-razavi/ITIC-paper#44]
@jrelliottoh @ramess101 

Something bugs about Eq.(6) as defined below

![image](https://user-images.githubusercontent.com/16358113/55518866-a60a4800-5643-11e9-9d96-1247d21138d8.png)

The above equation is only true if rho_1=0, so this equation is wrong by itself. The reviewer will probably complain about it. Why not reformulating this section as follows

![image](https://user-images.githubusercontent.com/16358113/55519001-26c94400-5644-11e9-91c8-3c28bec029da.png)

The above Eq.(6) is always true by itself even if rho_1 is not zero. We actually use this formula when numerically integrating along supercritical isotherm, so it has value by itself. Then we mention that in ITIC derivations we deal with rho1=0 along isotherms (supercritical isotherm and isotherm at Tsat on vapor side (Eq.8)) therefore LHS can be Adep(T,rho_2).

what do you guys think?

**Comment by ramess101** [comment]
@mostafa-razavi 

So should we change Equation 5 such that it is Adep(T1,rho) - Adep(T2,rho)? It wouldn't make sense to have one in terms of Adep and one as A. Would that be fine?

**Comment by ramess101** [comment]
@mostafa-razavi 

Is there a more general version of Equation 6 that is true even if rho1 != 0?

**Comment by mostafa-razavi** [comment]
> So should we change Equation 5 such that it is Adep(T1,rho) - Adep(T2,rho)? It wouldn't make sense to have one in terms of Adep and one as A. Would that be fine?

I think it makes sense because we are using Eq(5) as it is (in A form) in Eq(9) 


> Is there a more general version of Equation 6 that is true even if rho1 != 0?

**Comment by ramess101** [comment]
@mostafa-razavi 

OK, then if we need to keep Eq 5 as A form, I would keep Eq 6 in the A form and just explicitly include \rho1 =0 in A(T,rho1=0) and the integrand (rho1=0 to rho2). Would that be correct?

**Comment by mostafa-razavi** [comment]
> Is there a more general version of Equation 6 that is true even if rho1 != 0?

This equation might be correct: 

A(T,rho2)/RT - A(T,rho1)/RT = Int_{rho_1}^{rho_2} Z/rho drho

But we don't use this form anywhere, so I don't want to use it.

**Comment by mostafa-razavi** [comment]
> OK, then if we need to keep Eq 5 as A form, I would keep Eq 6 in the A form and just explicitly include \rho1 =0 in A(T,rho1=0) and the integrand (rho1=0 to rho2). Would that be correct?

That would be correct but it would lose its generality. On the other hand it would be consistent with Eq(5). I don't know

**Comment by ramess101** [comment]
> A(T,rho2)/RT - A(T,rho1)/RT = Int_{rho_1}^{rho_2} Z/rho drho

Yeah I was wondering if that was correct. It seems right since A \propto PdV so A/RT = PdV/RT or Z/rho drho... but so I am confused how we get the Z-1/rho form inside the integrand, shouldn't the -1 term come out after integrating with the lower bound as rho1=0?

**Comment by mostafa-razavi** [comment]
> ... A \propto PdV so A/RT = PdV/RT or Z/rho drho ....

How is PdV/RT = Z/rho drho? Shouldn't it be Z/drho rho?

**Comment by ramess101** [comment]
If V= 1/rho, then dV = -1/rho^2 drho.

Z = PV/RT or P/rhoRT so Z/rho = P/rho^2RT and thus -PdV/RT (A propto
negative PdV) is equal to Z/rho drho

**Comment by mostafa-razavi** [comment]
@ramess101 we need to decide between these:

1)
![image](https://user-images.githubusercontent.com/16358113/55518866-a60a4800-5643-11e9-9d96-1247d21138d8.png)

2)
![image](https://user-images.githubusercontent.com/16358113/55519001-26c94400-5644-11e9-91c8-3c28bec029da.png)

3)
![image](https://user-images.githubusercontent.com/16358113/55523488-909f1900-5657-11e9-867c-c4116ec3d87d.png)

**Comment by ramess101** [comment]
I think some feedback from Richard would be very helpful. Are each of these
equations valid? I am somewhat confused where the -1/rho term comes from
now.

**Comment by mostafa-razavi** [comment]
@jrelliottoh @ramess101 

I figured out an alternative path to derive the ITIC equations. The new path is longer but it's much simpler and more straightforward. Plus, it does not depend on different confusing types of departure function. I'm updating Section 2 and I'll send you the new manuscript when it's done.

**Comment by ramess101** [comment]
How different? I think it is risky to do a total rewrite at this point.
Make sure to save the old version
